_data/glossary/persistent-volume.yaml

```diff
@@ -0,0 +1,14 @@
+id: persistent-volume
+name: Persistent Volume
+full-link: /docs/concepts/storage/persistent-volumes/
+related:
+- statefulset
+- deployment
+- pod
+tags:
+- core-object
+- storage
+short-description: >
+  Persistent Volume is a kind of interface through two resource types, i.e. pv and pvc, and provides pod persistent storage service.
```

Comment:
Drop "Persistent Volume is" to avoid repeating the term being defined.

Reply:
Yup. Must have. To be explicit, short descriptions should be fragments, thus: `A kind of interface through ..."

I also dug around, and I can find no instance of "pv" or "pvc" in Kubernetes documentation that do NOT mean "persistent volume" and "persistent volume claim" respectively. (Note also that the acronym forms should be capitalized, PV and PVC).

Maybe the PersistentVolume paragraph in the Introduction to this topic can help: https://kubernetes.io/docs/concepts/storage/persistent-volumes/

Remember that complete novices to Kubernetes are the audience for the glossary, so if you can find a way to simplify the language it would be great.

Reply:
Suggestion: 
> An API object that represents a piece of storage in the cluster. Available as a general, pluggable resource that *persists* beyond the lifecycle of any individual {% glossary_tooltip term_id="pod" %}.